crds/insights-chart/PolicyReport_CRD.yaml

```diff
@@ -0,0 +1,334 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Must be of type `v1`

Reply:
We are migrating to v1  and testing with v1